config.json

```diff
@@ -0,0 +1,38 @@
+{
+    "EPIC": false,
+    "accessRequestFormLink": "https://example.com",
+    "api": {
+        "endpoint": "/api",
+        "host": "http://localhost"
+    },
+    "bannerExpirationDays": 14,
+    "cBio": false,
+    "demo": true,
+    "devUser": {
+        "token": "fb4d6830-d3aa-481b-bcd6-270d69790e11",
+        "user_id": "577cf6ef2b9920002cef0337"
```

Comment:
Are these private?

Reply:
No; these are only used in the demo environment.